### PR TITLE
Add space after inserted images

### PR DESCRIPTION
**Pull request by ebatty**
Necessary for jupyterbook layout fix

### CODE REVIEW

File: scripts/process_notebooks.py [hunk 1]
```diff
@@ -277,12 +277,12 @@ def extract_solutions(nb, nb_dir, nb_name):
 
                     tag_args = " ".join([
                         "alt='Solution hint'",
-                        "align='left'",
+                        "align='center'",
```
Comment (mwaskom) [inline scripts/process_notebooks.py:280]: Adding a space make sense by why change the alignment?

Comment (ebatty) [reply scripts/process_notebooks.py:280]: It seemed to need both to fix the layout issue (no idea why...)

Comment (mwaskom) [reply scripts/process_notebooks.py:280]: But it's going to change how the notebooks look on colab too, right? I think having the solution image centered and the actual image left-aligned will be annoying (I think we tried this last year and it looked bad).

Comment (ebatty) [reply scripts/process_notebooks.py:280]: Yeah it would... Let me see if I could just change for the jupyterbook

Comment (ebatty) [reply scripts/process_notebooks.py:280]: oh super easy to do by parsing the html just for jupyterbook I think. Will do that instead

**Pull request closed without merging**